lib/vdsm/virt/livemerge.py

```diff
@@ -402,6 +410,21 @@ def _start_commit(self, drive, job):
             self._untrack_job(job.id)
             raise exception.MergeFailed(str(e), job=job.id)
 
+    def _base_needs_prune_bitmaps(self, drive, job, base_info):
+        if base_info['format'] != 'COW':
+            log.debug("Base volume does not support bitmap operations "
+                      "job=%s drive=%s volume=%s",
+                      job.id, job.drive, job.base)
+            return False
+
+        if job.top == drive.volumeID:
+            log.debug("Top volume is the active volume "
+                      "job=%s drive=%s volume=%s",
```

Comment:
It would be nice to unify the way we log job and base. You copied the style use in _base_needs_extend() but in other code we log "job_id=%r", "base=%r" or "(job: %s)".

I think "job=%r base=%r" here will be better and later someone will unify the logs.

Reply:
How about this for both debug logs (regarding text & style)?

```python
    def _base_needs_prune_bitmaps(self, job, base_info):
        if base_info['format'] != 'COW':
            log.debug("Base volume does not support bitmaps, "
                      "job=%r base=%r", job.id, job.base)
            return False

        if job.active_commit:
            log.debug("Top volume is the active volume, "
                      "job=%r top=%s", job.id, job.top)
            return False
```